projects/uitpas/docs/errors.md

```diff
@@ -0,0 +1,341 @@
+
+## reward-period-constraint
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-period-constraint`
+* **Title**: `Reward Period Constraint`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the reward is not valid any more.
+
+## reward-volume-constraint
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-volume-constraint`
+* **Title**: `Reward Volume Constraint`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the reward is sold out.
+
+## reward-passholder-volume-constraint
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-passholder-volume-constraint`
+* **Title**: `Reward Passholder Volume Constraint`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the reward cannot be redeemed by this passholder anymore.
+
+## reward-passholder-points-constraint
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-passholder-points-constraint`
+* **Title**: `Reward Passholder Points Constraint`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the passholder does not have sufficient points to redeem this reward.
+
+## reward-passholder-no-applicable-cardsystems
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-passholder-no-applicable-cardsystems`
+* **Title**: `Reward Passholder No Applicable Cardsystems`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the passholder does not have a card system membership in an applicable card system.
+
+## reward-invalid-redeem-type
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/reward-invalid-redeem-type`
+* **Title**: `Reward Invalid Redeem Type`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the reward cannot be redeemed online.
+
+## checkin-code-expired
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/checkin-code-expired`
+* **Title**: `Check-in code expired`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the checkin code is expired.
+
+## checkin-code-invalid
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/checkin-code-invalid`
+* **Title**: `Check-in code invalid`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the checkin code is invalid.
+
+## checkin-not-allowed
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/checkin-not-allowed`
+* **Title**: `Check-in not allowed`
+* **Status**: `400`
+
+The request you are trying to perform cannot complete because the checkin is currently not allowed.
+
+## rate-limited
+
+* **Complete type:** `https://api.publiq.be/probs/uitpas/rate-limited`
+* **Title**: `Rate limited`
+* **Status**: `429`
+
+The request you are trying to perform cannot complete because it was performed too many times in a given period.
```

Comment:
This seems like something that might be better to add a generic error for in the future but okay to leave as-is for now

Reply:
could be an issue for backward compatibility - although clients probably should act on the HTTP code, not the error type in this case